pkg/tls/cred.go

```diff
@@ -12,9 +13,25 @@ import (
 )
 
 type (
+	// PrivateKeyEC wraps an EC private key
+	PrivateKeyEC struct {
```

Comment:
can `PrivateKeyEC` and `PrivateKeyRSA` be private?

Reply:
Good catch :+1: